recipes/monetdblite/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "monetdblite" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hannesmuehleisen/MonetDBLite-Python/archive/v{{ version }}.tar.gz
```

Comment:
Using the `.tar.gz` file from GitHub because unfortunately the author [does not publish sdists](https://pypi.org/project/monetdblite/#files).